### PR TITLE
Juicy features cargo installer wrapper

### PR DESCRIPTION
**Pull request by Azgrom**
Since I have the preference to checkout and install nushell directly from this repository, it has been kind of an annoyance to always check the nushell's book for the case there has been any change on the command to install all juicy stuff.

So I would like to propose having this script here, where it would always be updated accordingly.

### CODE REVIEW

File: extra_features_cargo_install.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+cargo install --path . --force --features=extra
```
Comment (jonathandturner) [inline extra_features_cargo_install.sh:1]: With recent `cargo`, you shouldn't need the `--force` flag any more. Do you see errors without it?

Comment (Azgrom) [reply extra_features_cargo_install.sh:1]: Nope. In fact I always use the `cargo install nu --features=extra`. Just copied and pasted the above mentioned command from the book.

If you wish, I would totally remove it.

Comment (jonathandturner) [reply extra_features_cargo_install.sh:1]: yeah, let's go ahead and remove it and I'll remove it from the book.

**Comment by jonathandturner** [comment]
Thanks!

**Pull request merged**